loop/constants.js

```diff
@@ -14,6 +14,23 @@ module.exports = {
     CONNECTED: "connected",
     TERMINATED: "terminated"
   },
+  ROOM_STATES: {
+    INIT: "init",
```

Comment:
Please, use the same `"` or `'` everywhere for consistency (at least in the PR. We don't enforce this at the project level)